src/storekit.cs

```diff
@@ -303,10 +307,12 @@ interface SKProduct {
 		string ProductIdentifier { get; }
 
 #if MONOMAC
```

Comment:
Same.

Reply:
Similarly, I don't this one can be removed either since the #if and the #else contain the same method names